change/@microsoft-teams-js-6364da81-f8f2-4864-b4b8-305ef45c6548.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "minor",
+  "comment": {
```

Comment:
This isn't the way the comment field is normally formatted. Should just be "comment": "Your comment here"